aspnetcore/fundamentals/metapackage.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Microsoft.AspNetCore.All metapackage for ASP.NET Core 2.X and later
+author: Rick-Anderson
+description: Microsoft.AspNetCore.All metapackage includes all supported packages.
+keywords: ASP.NET Core, NuGet, package, Microsoft.AspNetCore.All, metapackage
+ms.author: riande
+manager: wpickett
+ms.date: 07/16/2017
+ms.topic: article
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: fundamentals/metapackage
+---
+
+#Microsoft.AspNetCore.All metapackage for ASP.NET Core 2.X
+
+This feature requires ASP.NET Core 2.X.
```

Comment:
The "X" in 2.X is uppercase here, but it's lowercase in the tabs. Let's settle on one format or the other for the sake of consistency.